.coafile

```diff
@@ -1,18 +1,9 @@
-[Default]
-files = *.py, coalib/**/*.py, ./coala, tests/**/*.py, docs/conf.py
-ignore =
-    tests/bearlib/languages/documentation/documentation_extraction_testdata/*.py,
-    tests/collecting/collectors_test_dir/bears/incorrect_bear.py,
-
-max_line_length = 80
-use_spaces = True
-
 [python]
 # Patches may conflict with autopep8 so putting them in own section so they
 # will be executed sequentially; also we need the LineLengthBear to double
 # check the line length because PEP8Bear sometimes isn't able to correct the
 # linelength.
-bears = SpaceConsistencyBear, QuotesBear
```

Comment:
How are these changes relevant ? Please explain.

Reply:
I didnt mean to change .coafile but it somehow changed when i ran coala.

Reply:
You don't even need to push `.coafile` in your PR. Please delete this file from commit and the push again.